### ISSUE TITLE
Binary wheels for musl aarch64.

### ISSUE THREAD

**Issue by bigcat88** [python-greenlet/greenlet#292]
Subj. Can we get binary wheels for poor Alpine users on arm8 CPUs?
Is this feature on a roadmap or not?(in other words: should we expect to see it in a near future or not)
Thank you.

**Comment by jamadden** [comment]
I have no plans to do that, but it may be as easy as adding `musllinux_1_1_aarch64` to the list in `.github/workflows/tests.yml` and enabling emulation with a step:
```yaml
      - name: Enable emulation
        run: |
          docker run --rm --privileged hypriot/qemu-register
```

I'd be happy to look at a PR that does this when I get a chance.

**Comment by jamadden** [comment]
These are available in 1.1.3.post0.
